functions/common/methods.ts

```
@@ -457,50 +473,111 @@ export const getPayloadAuthorization = async (event: any): Promise<any> => {
+    } catch (err: any) {
+      console.error('JWT verification failed:', err)
+      const reason = err.code === 'ERR_JWT_EXPIRED' ? 'expired' : 'invalid'
+      return { isValid: false, reason, profile: null }
     }
   }
 
   switch (authorizationType) {
     case 'basic':
       authorizationString = authorizationString.substring(basic.length)
-      authProfile = await getBasicAuthProfile(authorizationString)
+      const basicProfile = await getBasicAuthProfile(authorizationString)
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)